docs/examples/auto-instrumentation/README.rst

```diff
@@ -192,7 +196,13 @@ similar to the following example:
             "http.status_code": 200
         },
         "events": [],
-        "links": []
+        "links": [],
+        "resource": {
+        "telemetry.sdk.language": "python",
+        "telemetry.sdk.name": "opentelemetry",
+        "telemetry.sdk.version": "0.16b1",
+        "service.name": ""
```

Comment:
```suggestion

```

Reply:
@lzchen , this change is not done by me. it comes from the init_tracing code in components.py :
```
def init_tracing(
    exporters: Sequence[SpanExporter], ids_generator: trace.IdsGenerator
):
    service_name = get_service_name()
    provider = TracerProvider(
        resource=Resource.create({"service.name": service_name}),
        ids_generator=ids_generator(),
    )
    trace.set_tracer_provider(provider)
```

Reply:
The outputs should be the same for auto-instrumented and manually instrumented. Since you removed `service_name` from appearing in the console export, it shouldn't appear for either example.

Reply:
@lzchen , there was mistake in that output snapshot in [previous comment](https://github.com/open-telemetry/opentelemetry-python/pull/1435#discussion_r534994259), the service.name was coming irrespective of my change. 

The change I made adds an additional "service_name" to the output (please see the output) ( which i removed as per your suggestion)
![auto-instru-example1](https://user-images.githubusercontent.com/55284676/101504757-99ea9800-3999-11eb-9a4c-59d8ba06bbc3.PNG)

The code in the init_tracing **hasnt been** touched by me for any of the changes I did.

My apologies that i realised it very late that output in the [previous comment](https://github.com/open-telemetry/opentelemetry-python/pull/1435#discussion_r534994259) was incorrect.